packages/libs/src/sdk/api/generated/default/apis/TracksApi.ts

```diff
@@ -252,6 +323,38 @@ export class TracksApi extends runtime.BaseAPI {
         return await response.value();
     }
 
+    /**
+     * @hidden
+     * Inspect an mp3 track including Content-Length in the response.
+     * Gets the availability of a streamable MP3 file of a track,
+     */
+    async inspectTrackRaw(params: InspectTrackRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<void>> {
+        if (params.trackId === null || params.trackId === undefined) {
+            throw new runtime.RequiredError('trackId','Required parameter params.trackId was null or undefined when calling inspectTrack.');
+        }
+
+        const queryParameters: any = {};
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        const response = await this.request({
+            path: `/tracks/{track_id}/stream`.replace(`{${"track_id"}}`, encodeURIComponent(String(params.trackId))),
+            method: 'HEAD',
+            headers: headerParameters,
+            query: queryParameters,
+        }, initOverrides);
+
+        return new runtime.VoidApiResponse(response);
+    }
+
+    /**
+     * Inspect an mp3 track including Content-Length in the response.
+     * Gets the availability of a streamable MP3 file of a track,
```

Comment:
Nit: The description field from the decorator of the method in Python is merged with the function description (second line and on). The summary is added after. It's a bit weird